### ISSUE TITLE
Is there a way to redirect the user to the page they are already on?

### ISSUE THREAD

**Issue by 684efs3** [pbteja1998/remix-auth-google#19]
I would like the user to remain on their current page rather than be redirected to a dashboard or profile page. Is this possible? Thank you!

**Comment by cluk3** [comment]
Yes, as you have access to `request.url` you can simply redirect to `new URL(request.url).pathname`
